critiquebrainz/frontend/views/artist.py

```diff
@@ -46,3 +48,56 @@ def entity(id):
         release_group['review_count'] = review_count
     return render_template('artist.html', id=id, artist=artist, release_type=release_type,
                            release_groups=release_groups, page=page, limit=limit, count=count)
+
+
+def squash_duplicated_members(members):
```

Comment:
I would add a docstring here, describe what's going on and what potentially confusing stuff like `target` is.